tests/testsuite.src/data_pointer.at

```diff
@@ -44,18 +44,18 @@ AT_CHECK([test "$COB_HAS_64_BIT_POINTER" = "yes"], [0], [], [],
 
 # Previous test "failed" --> 32 bit
 
-[AT_CHECK([$COBCRUN_DIRECT ./prog], [0], [0x00000000
```

Comment:
This is using a format we commonly don't use, a conditional when a test "passed"/"failed". The [] scope these parts, so they should be kept in..

Reply:
Now, these `[...]` scopes are mandatory for my tool, so I actually had to add them in a few places.